src/platforms/browser/WebPlatform.mjs

```diff
@@ -112,6 +113,109 @@ export default class WebPlatform {
         return cancelCb;
     }
 
+    createRoundRect(cb, stage, w, h, radius, strokeWidth, strokeColor, fill, fillColor) {
+        if (fill === undefined) fill = true;
+        if (strokeWidth === undefined) strokeWidth = 0;
+
+        let canvas = this.getDrawingCanvas();
```

Comment:
Prefer const over let

Reply:
Moved this code back to Tools.mjs based on feedback.  This code is now left as is (and unmodified) in Tools.mjs.